docs/advanced-usage/connecting-cloud-storage/google-cloud-storage.md

```diff
@@ -1,3 +1,9 @@
+---
+icon: simple/googlecloud
```

Comment:
Where does this icon come from? (/ where can I find it?)

Reply:
Interesting. I guess several icon sets come bundled here. Good to know https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/?h=icons